src/Core/Application/Identity/Roles/PermissionDto.cs

```diff
@@ -4,4 +4,5 @@ public class PermissionDto
 {
     public string? Permission { get; set; }
     public string? Description { get; set; }
+    public bool Enabled { get; set; }
```

Comment:
don't see the point of this...

Reply:
Now, is only being used in the Client Side, i kept it in here so is the same in future client apps. 
You think is better create  a new Dto only for the Client App ?

Reply:
Yes, definitely!